Sources/Client/State/StartupState.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import DeltaCore
+
+enum StartupState {
+  case loading
```

Comment:
Did you mean to delete LoadingState?

Reply:
No, check response here
https://github.com/stackotter/delta-client/pull/41#discussion_r744136882

Reply:
ah ok, my bad